### PR TITLE
Support region split and scatter api

### PR DESCRIPTION
**Pull request by Yisaer**

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -530,10 +532,14 @@ message ScatterRegionRequest {
...
     // If not defined, the regions would be scattered in a cluster level.
     string group = 5;
+
+    repeated uint64 regions_id = 6;
+    uint64 retry_limit = 7;
 }
 
 message ScatterRegionResponse {
     ResponseHeader header = 1;
+    uint64 finishedPercentage = 2;
```
Comment (HunDunDM) [inline proto/pdpb.proto:542]: Illegal format

File: proto/pdpb.proto [hunk 2]
```diff
@@ -633,6 +639,18 @@ message SyncMaxTSResponse {
...
 
+message SplitRegionsRequest {
+    RequestHeader header = 1;
+    repeated bytes split_keys = 2;
+    uint64 retry_limit = 3;
+}
+
+message SplitRegionsResponse {
+    ResponseHeader header = 1;
+    uint64 finishedPercentage = 2;
```
Comment (HunDunDM) [inline proto/pdpb.proto:650]: ditto

File: proto/pdpb.proto [hunk 3]
```diff
@@ -530,10 +532,14 @@ message ScatterRegionRequest {
     // If group is defined, the regions with the same group would be scattered as a whole group.
     // If not defined, the regions would be scattered in a cluster level.
     string group = 5;
+
+    repeated uint64 regions_id = 6;
```
Comment (HunDunDM) [inline proto/pdpb.proto:536]: Please add the relationship between it and `region_id`.

File: proto/pdpb.proto (hunk 1 above)
Comment (HunDunDM) [reply proto/pdpb.proto:542]: This is not a stream request, please add a comment on how to notify multiple times when introducing `percentage`.

File: proto/pdpb.proto (hunk 3 above)
Comment (HunDunDM) [reply proto/pdpb.proto:536]: If there are some fields that need to be deprecated, please refer to the [documentation](https://developers.google.com/protocol-buffers/docs/proto3#options) to mark them.

Comment (Yisaer) [reply proto/pdpb.proto:537]: updated.

**Comment by rleungx** [comment]
PTAL @HunDunDM

**Review by HunDunDM — APPROVED** [review]

**Review by rleungx — APPROVED** [review]

**Pull request merged**